django/river/settings/base.py

```diff
@@ -0,0 +1,216 @@
+"""
+Django settings for river project.
+
+Generated by 'django-admin startproject' using Django 3.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
+
+# API versioning
+VERSION_NAME = os.environ.get("VERSION_NAME", None)
+VERSION_SHA = os.environ.get("VERSION_SHA", None)
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get("SECRET_KEY")
```

Comment:
does django need a secret key ? I don't remember having to use a secret key in river before

Reply:
The secret key is used to encrypted sensible data in the DB. We don't use the secret key (yet). It's unpractical to disable and we will be hopefully using it soon, so I've let this in.